<!-- launch file for Yonah's Iridium Short-Burst-Data (SBD) service (for testing purposes)-->

<launch>

    <!-- screen = output to roslaunch screen for debug; log = log to rosout-->
    <arg name="log_output" default="screen"/>

    <!-- sbd link node -->
    <node pkg="sbd" type="sbd_link.py" name="sbd_link"
    respawn="true" respawn_delay="3" output="$(arg log_output)">
    </node>

    <!-- satcomms test node -->
    <node pkg="sbd" type="satcomms_test.py" name="satcomms_test"
    respawn="true" respawn_delay="3" output="$(arg log_output)">
    </node>

</launch>
